course_viewer/course_viewer/models.py

```diff
@@ -0,0 +1,111 @@
+from django.db import models
+
+
+class State(models.Model):
+    '''
+    The Transfer State
+    '''
+    state_code = models.CharField(
+                    max_length=2,
+                    unique=True,
+                    help_text='The abbreviation of the transfer college state')
+
+    state_name = models.CharField(
+                    max_length=200,
+                    unique=True,
+                    help_text='The long name of the transfer college state')
+
+    def __str__(self):
+        return self.state_name
+
+    class Meta:
+        ordering = ['state_code']
+        verbose_name = 'state'
+        verbose_name_plural = 'states'
+        db_table = 'states'
+
+
+class College(models.Model):
+    '''
+    The Transfer College
+    '''
+    college_code = models.CharField(
+                    max_length=15,
+                    primary_key=True,
+                    help_text='The code of the transfer college')
+    college_name = models.CharField(
+                        max_length=50,
+                        help_text='The name of the transfer college')
+
+    def __str__(self):
+        return self.college_name
+
+    class Meta:
+        ordering = ['college_code']
+        verbose_name = 'college'
+        verbose_name_plural = 'colleges'
+        db_table = 'colleges'
+
+
+class Course(models.Model):
+    '''
+    The Transfer Course
+    '''
+    transfer_course_id = models.CharField(
+                        max_length=15,
+                        help_text='The id of the transfer course')
+    transfer_course_credit = models.IntegerField(
+                    help_text='The number of credits the transfer course is')
```

Comment:
Some of the entries don't have values for `transfer_course_credit`, they just are equivalent to a class (annoyingly enough without boosting credits taken).  Will Django reflect this properly with a `None` type, or will this cause an issue?

Reply:
Looking farther into this actually, it's the same half credit point issue as the MTU credit side annoyingly enough. `International Institute` has several half credit classes.

Reply:
I think 
`blank=True`
and using a Float field would be best here in that case